Capstone/Detail View/DetailsViewController.m

```diff
@@ -58,6 +58,17 @@ - (void)handleDoubleTap:(UITapGestureRecognizer *)sender {
     [self performSegueWithIdentifier:@"moreDetails" sender:self];
 }
 
+- (IBAction)didTapDirectionsToCollege:(id)sender {
+    CLLocationCoordinate2D coordinate = [self getLocation];
+    CLLocationCoordinate2D start = {coordinate.latitude, coordinate.longitude};
+    CLLocationCoordinate2D destination = {[self.college.lat doubleValue], [self.college.longtuide doubleValue]};
+
```

Comment:
nit: extra space

Reply:
controversial after all the feedback we've been giving, but i think we're at a point where we can add some nuance around spacing, given all the good work at resolving these nits have been!

we mainly don't want extra spacing in between } & {:
```
// end of function A
}


- (void)someFunction {
```

and we don't want extra spacing at the top and bottom of braces:
```
- (void)someFunction {

int index = 0;
}
```
```
- (void)someFunction {
int index = 0;

}
```
```
if (something) {

int index = 0;
}
```

*however*, sometimes it makes sense to have single newlines in between groupings *within* a method: e.g. having a group for variables, and a group for how they're used; or having a group doing action A, then another group doing action B

Reply:
I see! That makes sense. However, I find it kind of odd that there was no extra space on my IDE, but there's an extra space on line 65 in github. Anyways, resolved!